docs/en/contribution/new-license.md

```diff
@@ -1,25 +1,37 @@
 # How To Add New License
 
-If you have any new Jar binary package adding in you PR, you need to follow the steps below to notice license
+### ASF 3RD PARTY LICENSE POLICY
 
-1. declared in `tools/dependencies/known-dependencies.txt`
+You have to pay attention to the following open-source software protocols which Apache projects support when you intend to add a new feature to the SeaTunnel (or other Apache projects), which functions refers to other open-source software references.
 
-2. Add the corresponding License file under `seatunnel-dist/release-docs/licenses`, if it is a standard Apache License, it does not need to be added
+[ASF 3RD PARTY LICENSE POLICY](https://apache.org/legal/resolved.html)
 
-3. Add the corresponding statement in `seatunnel-dist/release-docs/LICENSE`
+If the 3rd party software is not present at the above policy, we could't that accept your code.
 
-   ```bash
-   # At the same time, you can also use the script to assist the inspection.
-   # Because it only uses the Python native APIs and does not depend on any third-party libraries, it can run using the original Python environment.
-   # Please refer to the documentation if you do not have a Python env: https://www.python.org/downloads/
-   
-   # First, generate the seatunnel-dist/target/THIRD-PARTY.txt temporary file
-   ./mvnw license:aggregate-add-third-party -DskipTests -Dcheckstyle.skip
-   # Second, run the script to assist the inspection
-   python3 tools/dependencies/license.py seatunnel-dist/target/THIRD-PARTY.txt seatunnel-dist/release-docs/LICENSE true
-   ```
 
-4. Add the corresponding statement in `seatunnel-dist/release-docs/NOTICE`
+### How to Legally Use 3rd Party Open-source Software in the SeaTunnel
 
-If you want to learn more about strategy of License, you could read
-[License Notice](https://seatunnel.apache.org/community/submit_guide/license) in submit guide.
+Moreover, when we intend to refer a new software ( not limited to 3rd party jar, text, CSS, js, pics, icons, audios etc and modifications based on 3rd party files) to our project, we need to use them legally in addition to the permission of ASF. Refer to the following article:
+
+* [COMMUNITY-LED DEVELOPMENT "THE APACHE WAY"](https://apache.org/dev/licensing-howto.html)
+
+
+For example, we should contain the NOTICE file (every open-source project has NOTICE file, generally under root directory) of ZooKeeper in our project when we are using ZooKeeper. As the Apache explains, "Work" shall mean the work of authorship, whether in Source or Object form, made available under the License, as indicated by a copyright notice that is included in or attached to the work.
```

Comment:
😂 I think not every open source project has a notice file. Many non-Apache projects have no such file actually.

Reply:
my mistake thanks for your reminder